spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/RDDFunctions.scala

```diff
@@ -159,14 +158,39 @@ class RDDFunctions[T](rdd: RDD[T]) extends WritableToCassandra[T] with Serializa
   implicit
     connector: CassandraConnector = CassandraConnector(sparkContext.getConf),
     currentType: ClassTag[T],
-    rwf: RowWriterFactory[T]) = {
+    rwf: RowWriterFactory[T]): CassandraPartitionedRDD[T] = {
+
+    val converter = ReplicaMapper[T](connector, keyspaceName, tableName, partitionKeyMapper)
```

Comment:
Yeah i'll try to rename this, this is the factory that uses the Java Driver to transform the Partition key into a set of Ip addresses. So maybe ReplicaLocator ?